### PR TITLE
continue-on-error-composite-fix

### PR DESCRIPTION
**Pull request by jablonnc**
Addresses the [following error](https://github.com/JupiterOne/web-query-results/actions/runs/5979730248/job/16224420429). For some reason we have to type cast the `continue-on-error` when it's being used with a composite action.

### CODE REVIEW

**Review by Sivli-Embir — APPROVED** [review]

**Review by Sivli-Embir — APPROVED** [review]

File: .github/workflows/docs/frontend/frontend_npm_release.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ jobs:
     uses: jupiterone/.github/.github/workflows/frontend_npm_release.yml
     secrets:
       NPM_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      AUTO_GITHUB_PAT_TOKEN: ${{ secrets.AUTO_GITHUB_PAT_TOKEN }}
```
Comment (ImDevinC) [inline .github/workflows/docs/frontend/frontend_npm_release.md:40]: These changes seem unrelated?

Comment (jablonnc) [reply .github/workflows/docs/frontend/frontend_npm_release.md:40]: They are, just missed a prop on the documentation

**Review by ImDevinC — APPROVED** [review]
